notebooks/05_operationalize/lgbm_webservice_poc.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<p>is this LigthGBM?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Yes. But used in an item similarity fashion.